tests/unit/database_tests.py

```diff
@@ -202,6 +202,11 @@ def test_create_db_delete_db(self):
             # No issue should arise if attempting to create existing database
             db_2 = db.create()
             self.assertEqual(db, db_2)
+            # If we use throw_on_exists=True, it will arise a
+            # CloudantDatabaseException if the database already exists.
+            with self.assertRaises(CloudantDatabaseException) as cm:
+                db.create(throw_on_exists=True)
+            self.assertEqual(cm.exception.status_code, 100)
```

Comment:
I see. The point is: `self.r_session.put(self.database_url)` is not returning a 409 error, but 412: 
<img width="499" alt="a" src="https://cloud.githubusercontent.com/assets/1151470/23760357/329c3b9e-04ce-11e7-8cb7-066c61acc7cc.png">

Since there are no 412 code on `_messages.py`, it is creating a `CloudantDatabaseException` with code 100. We can handle this creating a new 412 error on `DATABASES = { ... }` (_messages.py). 

I don't think a 409 would resolve the issue since this code is `Document with id _ already exists.`, and that's not the issue we should have, correct?